src/types/profile/settings.rs

```diff
@@ -26,7 +26,12 @@ pub struct Settings {
     pub subtitles_text_color: String,
     pub subtitles_background_color: String,
     pub subtitles_outline_color: String,
+    /// Whether or not the Escape key should exists from the app when in Full screen.
+    pub esc_exists_fullscreen: bool,
```

Comment:
Should be `esc_exit_fullscreen`

Reply:
Fixed.

Reply:
It should be `esc_exit_fullscreen` not `esc_exist_fullscreen`

Reply:
lol, fixed the typo,again :D